sdk/metric/manual_reader.go

```diff
@@ -118,8 +118,8 @@ func (mr *ManualReader) Shutdown(context.Context) error {
 	return err
 }
 
-// Collect gathers all metrics from the SDK and other Producers, calling any
-// callbacks necessary and stores the result in rm.
+// Collect gathers and returns all metric data related to the Reader from
```

Comment:
This doesn't return any metric data. That data is stored to the passed `rm`.

Reply:
PTAL df916862